apps/meteor/client/views/admin/apps/BundleChips.tsx

```diff
@@ -15,19 +15,22 @@ type BundleChipsProps = {
 
 const BundleChips = ({ bundledIn }: BundleChipsProps): ReactElement => {
 	const t = useTranslation();
-
+	const bundleRef = useRef<Element>();
 	return (
 		<>
-			{bundledIn.map((bundle, index) => (
-				<Tag
-					key={index}
-					title={t('this_app_is_included_with_subscription', {
-						bundleName: bundle.bundleName,
-					})}
-					variant='primary'
-				>
-					{bundle.bundleName}
-				</Tag>
+			{bundledIn.map((bundle) => (
+				<Fragment key={bundle.bundleId}>
+					<Box ref={bundleRef}>
```

Comment:
I didn't get the point of this ref 🥲
Is it doing something?

Reply:
Not right now, in the past it connected a tooltip to the bundle tag. I'll fix this.

Reply:
Solved